nise/yaml_generators/oci/generator.py

```diff
@@ -68,7 +57,7 @@ def generate_tags(config, key):
 
     tags = []
     if not config.get("tags"):
-        keys = TAG_KEYS.get(key)
+        keys = OCIYamlConstants.get_tag_keys(key)
```

Comment:
This is being called like a static method but isn't defined as such. I find it interesting that this works. 🤔

Reply:
I see what's happening. This is working by accident.

Instance methods implicitly accept a single parameter, which is the instance of `OCIYamlConstants`. When called as a static method, there is no instance object, so the first parameter is `key`. If this is called on an instance of the object, a `TypeError` is raised because two args are passed to the method.

```
In [2]: OCIYamlConstants().get_tag_keys('storage')
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In [2], line 1
----> 1 OCIYamlConstants().get_tag_keys('storage')

TypeError: get_tag_keys() takes 1 positional argument but 2 were given
```

Reply:
moved `get_tag_keys() ` outside the class